src/lib/markdown.ts

```diff
@@ -118,6 +120,21 @@ const anchorMarkedExtension = {
   },
 };
 
+export const walkTokens = (token: marked.Tokens.Generic) => {
+  if (token.type !== "code" && token.type !== "codespan") {
+    if (
+      "text" in token &&
+      !token.text.match(new RegExp(`${trustedHtmlTags.join("|")}`, "g"))
+    ) {
+      token.text = token.text.replace(
+        /<(.*)>/g,
+        (_match: RegExpMatchArray, tagContent: string) =>
+          `&lt;${tagContent}&gt;`,
```

Comment:
Would escaping work? ie. `\<` and `\>`?

Reply:
I tried this first, but it ended up rendering the backslashes instead of escaping the `<` and hiding the backslash